frank/plot.py

```diff
@@ -20,15 +20,21 @@
 Frankenstein fits.
 """
 
-from frank.utilities import sweep_profile
-
 import numpy as np
 import matplotlib.pyplot as plt
 
+# Suppress some benign warnings
+import warnings
+warnings.filterwarnings('ignore', '.*compatible with tight_layout.*')
+warnings.filterwarnings('ignore', '.*handles with labels found.*')
```

Comment:
I don't like ignoring these warnings. Can you remove the need to use tight layout by specifying the plot geometry explicitly

Reply:
It's the most useful way to quickly format the figures across a range of backends used. It works well for the figures I produce, and I've made the warnings ignored strictly only these 2. Even if they were printed, they would not affect the code beyond the figure aesthetics. There won't be any missed important error messages, because only these 2 benign ones are ignored.

Reply:
Is it possible to place this in a with statement exactly where the calls are? Its best not to silence warnings globally